.betterer.eslint.config.js

```diff
@@ -128,6 +128,16 @@ module.exports = [
           selector: 'MemberExpression[object.name=localStorage]',
           message: 'Direct usage of localStorage is not allowed. Use `Store` from @grafana/data instead.',
         },
+        {
+          selector: 'JSXOpeningElement[name.name="Card"]:not(:has(JSXAttribute[name.name="noMargin"]))',
```

Comment:
This will apply to any component named Card, even if it's not from grafana/ui, but I'm not sure if there's a way to check for that.

Reply:
It could be done with a custom lint rule, but I don't think there's an "easy" way.